akka-camel/src/main/scala/akka/camel/Camel.scala

```diff
@@ -87,6 +87,8 @@ class CamelSettings private[camel] (config: Config, dynamicAccess: DynamicAccess
    */
   final val StreamingCache: Boolean = config.getBoolean("akka.camel.streamingCache")
 
+  final val ProducerChildDispatcher: String = config.getString("akka.camel.producer.use-dispatcher")
```

Comment:
Would perhaps be neat if this were an `Option[String]` (and make it `None` when the config string is empty), but this seems OK too :).